src/AndorSIF.jl

```diff
@@ -184,20 +190,27 @@ function FileIO.load(fs::Stream{format"AndorSIF"})
     for i = 1:frames
         _ = readline(io)
     end
-    offset = position(io) # start of the actual pixel data, 32-bit float, little-endian
 
-    pixels = read(io, Float32, width, height, frames)
+    # two additional lines?
+    _ = readline(io)
+    _ = readline(io)
+
+    offset = position(io) # start of the actual pixel data, 32-bit float, little-endian
+    numbytes = 4 * frames * width * height # number of bytes to read
+    pixelbytes = read(io, numbytes)
+    pixelarray = reinterpret(Float32, pixelbytes)
+    pixelmatrix = reshape(pixelarray, (width, height))
     prop = Compat.@compat Dict(
         "colorspace" => "Gray",
-        "spatialorder" => ["y", "x"],
+        "spatialorder" => ["x", "y"],
         "ixon" => ixon,
         "suppress" => Set{Any}(("ixon",)),
         "pixelspacing" => [1, 1]
     )
     if frames > 1
         prop["timedim"] = 3
     end
-    Image(float64(pixels), prop)
+    ImageMeta(pixelmatrix .|> Float64, prop)
```

Comment:
I don't see that it's important to return `Float64`, `Float32` seems entirely appropriate.